app/views/works/_work_activity_messages.html.erb

```diff
@@ -19,15 +37,21 @@
       </li>
     <% end %>
   </ul>
-
-  <div id="new-message-section">
-    <%= form_with url: add_message_work_path(@work) do |f| %>
-      <textarea id="new-message" name="new-message" class="new-message" placeholder="leave a message" rows="5" cols="30"></textarea>
-      <br/>
-      <%= f.submit("Message", class: "btn btn-secondary") %>
-      <span class="new-message-help">
-        Simple Markdown is accepted, e.g. *italics*, **bold**, # Header 1, ## Header 2, ```code```. Use @netid to refer to others.
-      </span>
-    <% end %>
-  </div>
 </div>
+
+<script>
```

Comment:
@hectorcorrea was there a reason you added this script?  Did the toggle button stop working?  It was working for me earlier in the week.

Reply:
The help text wrapped funny so I had added the toggle so that it does not look as funny.

(that text always wrapped funny, but since it was a the very bottom of the page it was not obvious, now that it is at the top it was more prominent)